app/views/escorts/_assessment.html.slim

```diff
@@ -12,7 +12,7 @@ section id=name.downcase
       p.review
         = t('assessment.confirmed_timestamp', name: assessment.reviewer.full_name, time: assessment.reviewed_at.to_s(:humanized))
 
-  - if assessment
+  - if assessment && assessment.any_questions_relevant?
```

Comment:
usually rubocop complains about not using & operator
assessment&.any_questions_relevant?